pyanaconda/ui/gui/spokes/blivet_gui.py

```diff
@@ -205,6 +205,7 @@ def _do_check(self):
 
         return self._error == ""
 
+    # TODO: this function is never used
```

Comment:
@vojtechtrefny what do you think?

Reply:
This function is used from blivet-gui -- https://github.com/storaged-project/blivet-gui/blob/master/blivetgui/osinstall.py#L182

It's a not-so-nice hack -- these buttons are part of the spoke but are actually controlled by blivet-gui.

Reply:
Kept.